<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="judgeButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>judge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getTheatrePlayInfoButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>get theatre and play info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrangeButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>arrange</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inputFromNewScheduleButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>input from new schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="displayButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>display to schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveToNewPlayButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>save to new play</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="dayStatusBrowser">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>141</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="gapResultBrowser">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="judgeResultBrowser">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>561</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
